overlay.d/05rhcos/usr/lib/systemd/system/rhcos-fail-boot-for-older-luks-syntax.service

```diff
@@ -0,0 +1,15 @@
+# Fixes the encryption issue with LUKS in 4.7 when a user 
+# tries to use the RHCOS 4.6 or earlier LUKS syntax in an 
+# Ignition config.
+[Unit]
+Description=Fail boot if Ignition uses an older syntax for enabling LUKS in 4.7 
+Before=ignition-disks.service
+After=ignition-fetch.service
```

Comment:
It looks like this is intending to run in the initramfs, but it's actually running in the real root.  It will need to become a dracut module (like some of the other ones) to make that work.